src/components/HorizontalCard/index.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react'
+
+import styles from './HorizontalCard.module.scss'
+
+import classNames from 'classnames'
+
+export interface HorizontalCardProps {
```

Comment:
same comment as here https://github.com/HyperPlay-Gaming/hyperplay-ui/pull/532#discussion_r2190814222

```suggestion
export interface HorizontalCardProps extends React.HTMLAttributes<HTMLDivElement> {
```